rustfmt.toml

```diff
@@ -1 +1,3 @@
-max_width = 600
+max_width = 1000
```

Comment:
That's very… unorthodox.
Wouldn't it be better to break all those long string literals into shorter lines instead and make the width `100` (that's default anyway, I think)?